federation/pkg/federation-controller/util/federated_informer.go

```diff
@@ -45,6 +45,9 @@ type FederatedReadOnlyStore interface {
 	// Returns all items in the store.
 	List() ([]interface{}, error)
 
+	// Returns all items from a cluster.
+	ListCluster(clusterName string) ([]interface{}, error)
```

Comment:
I don't like "ListCluster" name - it's very misleading.

Maybe "ListFromCluster" ?